class/defaults.yml

```diff
@@ -6,56 +6,112 @@ parameters:
     namespace: syn-${_instance}
     namespaceMetadata: {}
 
-    host_service_cidr: null
+    charts:
+      vcluster:
+        source: https://charts.loft.sh
+        version: 0.24.1
 
     images:
       k3s:
         registry: docker.io
         image: rancher/k3s
-        tag: v1.25.15-k3s1
+        tag: v1.32.1-k3s1
       syncer:
-        registry: docker.io
-        image: loftsh/vcluster
-        tag: "0.14.1"
+        registry: ghcr.io
+        image: loft-sh/vcluster-pro
+        tag: ""
```

Comment:
We should use a defined version to not just randomly break stuff when evicting vcluster pods.

Reply:
I've set the version specifically to the latest version